src/elements/fields/RadioButtonGroupField.jsx

```diff
@@ -19,6 +26,14 @@ function RadioButtonGroupField({
   const [otherSelect, setOtherSelect] = useState({});
   const otherChecked =
     (otherSelect[servar.key] || fieldVal) && fieldVal === otherVal;
+
+  const styles = useMemo(() => {
+    applyCheckboxStyles(element, applyStyles);
```

Comment:
Can we call this something more generic? A little confusing when it looks like youre appliyng checkbox styles to a radio group